src/d2q9_inc/Dynamics.R

```diff
@@ -0,0 +1,50 @@
+# Density - table of variables of LB Node to stream
```

Comment:
I know that we don't have such a thing in other models (to be filled), but it's good time to start:

Add something like "# Maintainer: Michal Dzikowski @mdzik" So we know who's model is it (as we really get lot of models now.

We should include this in #34 #36